Jenkinsfile

```diff
@@ -73,6 +73,7 @@ tests/mpi_test.sh'''
             source /opt/intelFPGA_pro/19.1/hld/init_opencl.sh
             export PATH=~/.local/bin:$PATH
 export DACE_debugprint=1
+export DACE_compiler_default_data_types=C
```

Comment:
Hmm, I don't want to touch the tests, especially since the issue manifests only with Intel's FPGA toolchain. The problem is quite simple. The default value for the flag is `Python`, i.e. integral data are interpreted to be int64. This leads to generating code that calls an Intel FPGA API method with `long long` arguments. However, this method only supports `int` parameters. Perhaps @definelicht and @TizianoDeMatteis have better insight.

Reply:
Can we set it just for Intel tests?

Reply:
This is what is currently being done (through the Jenkins configuration file). I think @tbennun meant to set this somehow specifically in the actual test (py) files.

Reply:
ok, we can keep it because it's an architecture requirement. IMO I think we should just add dtypes to `dace.symbol` definitions in the samples that break.